### ISSUE TITLE
Issue in Change Relationship "hreflang": 2.7.1 plugin release

### ISSUE THREAD

**Issue by MikeSmir** [inpsyde/MultilingualPress#280]
#### Version Information

* PHP: 5.4
* WordPress: 4.8.1
* Plugin release 2.7.1

#### Steps to Reproduce

Create Relationship between several posts (different sub-folder websites /en/, /de/, /ru/ and etc.)
1. /en/ to /de/
2. /ru/ to /de/ and /en/

#### What I Expected

The hreflang will create relationship between these websites one by one.

#### What Happened Instead
1. /en/ to / de/ will create the relationship.
2. /ru/ to /de/ will create the relationship, BUT delete for /en/...

I was trying to create relationship between 5 different language pages multiple times, and only two of them could be linked at the same time.

Rolled back to plugin release 2.7.0, everything is working great.

Thanks

**Comment by Dinamiko** [comment]
Hi MikeSmir,

I’m not able to reproduce the issue, can you explain a bit more the steps about how do you end up with a post that is not related at all anymore?

The only thing that we changed regarding relationship in 2.7.1 release was that the whole relationship data now is deleted when you remove the last relation. see issue #273.

Thanks,
Emili

**Comment by tfrommen** [comment]
Hi @MikeSmir,

any news on the issue?

My colleague and I are unable to reproduce this.
I tried both the advanced and the simple translation interface, and it just works like before.

Can you tell us what exactly you did how and where, or maybe even do a quick video/screencast?

Thanks in advance,
Thorsten

**Comment by MikeSmir** [comment]
Hi, 
Sorry for not answering, busy at work.

I think, what I was doing, I deleted the relationship between two pages and then was trying to link them again. On this stage I was having problems. 

What now:
I reinstalled the plugin v2.7.1, and it links my pages from the main website to sub-folder websites OK, but if I try to link, for example from /es/ Spanish website to the main website, it won't work, it doesn't create the relationship.

But, it's OK, I can work with it. 

I will try to install my website network onto the virtual machine and play with it. Don't want to test my online one. 

Thanks,
Mike
